pkg/network/ebpf/c/protocols/classification/dispatcher-helpers.h

```diff
@@ -75,15 +59,14 @@ static __always_inline void classify_protocol_for_dispatcher(protocol_t *protoco
         *protocol = PROTOCOL_UNKNOWN;
     }
 
-    log_debug("[protocol_dispatcher_classifier]: Classified protocol as %d %d; %s\n", *protocol, size, buf);
+//    log_debug("[protocol_dispatcher_classifier]: Classified protocol as %d %d; %s\n", *protocol, size, buf);
```

Comment:
why?

Reply:
If I include it, the code won't compile because of BPF stack limit (512 bytes)